composer.json

```diff
@@ -22,19 +22,21 @@
     ],
     "require": {
         "php": ">=7.1",
-        "bedita/i18n": "^1.6",
-        "bedita/web-tools": "^1.7.1",
+        "bedita/i18n": "^1.7",
+        "bedita/web-tools": "^1.8",
+        "cakephp/cakephp": "^3.10",
```

Comment:
since in `bedita/web-tools` 1.8 we already require `"cakephp/cakephp": "3.10.*"` I don't think the `cakephp/cakephp` dependency is necessary

Reply:
not necessary, but a good way to have a version specified in main app (@batopa suggested an explicit dependency here)

Reply:
When an item of a dependency is used in the main code base is a best practice to explicit the dependency in `composer.json`. This makes more clear what are the dependencies of the app and give more control on specific version used.